packages/xo-server-sdn-controller/src/index.js

```diff
@@ -738,6 +759,25 @@ class SDNController extends EventEmitter {
         this._starCenters.set(newCenter.$id, newCenter.$ref)
       }
     }
+
+    for (const poolNetwork of this._poolNetworks) {
+      const tunnel = this._getHostTunnelForNetwork(host, poolNetwork.network)
```

Comment:
If you decide to pass the network ref instead, you need to change it in `_getHostTunnelForNetwork` as well and make sure all the calls to the method respect the new signature.